src/lib/components/End.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+	import { t } from 'svelte-i18n';
+	import { gameStore } from '$lib/stores/gameStore';
+</script>
+
+<h1>{$t('game-over')}</h1>
+<button on:click={gameStore.startGame}>{$t('replay')}</button>
+<button on:click={gameStore.reset} style="margin-top: 1rem;">
```

Comment:
Button styling should be in `<style>` tags instead of inline. I recommend putting both buttons under one div instead of doing separate styling based on ID or something.